swagger/widgets/widgets_and_boards.json

```diff
@@ -0,0 +1,596 @@
+            "allOf": [
+              {
+                "$ref": "#/components/schemas/ComponentDataDto"
+              }
+            ]
+          }
+        },
+        "required": []
+      },
+      "MultiWidgetsResponse": {
+        "type": "object",
+        "properties": {
+          "widgets": {
+            "type": "array",
+            "items": {
+              "$ref": "https://vcita.github.io/developers-hub/entities/dashboard/widget.json"
+            }
+          }
+        },
+        "required": [
+          "widgets"
+        ]
+      },
+      "StaffWidgetsBoardRequestDto": {
+        "type": "object",
+        "properties": {
+          "board_layout": {
+            "type": "string",
+            "description": "The widgets board layout"
+          },
+          "type": {
+            "type": "string",
+            "description": "The board type"
+          },
+          "sections": {
+            "description": "The id of the number the call is from",
+            "allOf": [
+              {
+                "$ref": "#/components/schemas/SectionDto"
+              }
+            ]
+          }
+        },
+        "required": [
+          "staff_uid",
+          "client_uid",
+          "from_number",
+          "to_number",
+          "direction",
+          "status",
+          "rate",
+          "provider",
+          "source_id",
+          "external_uuid",
+          "business_uid"
+        ]
+      },
+      "MultiStaffWidgetsBoardResponseDto": {
+        "type": "object",
+        "properties": {
+          "staffWidgetsBoards": {
+            "type": "array",
+            "items": {
+              "$ref": "https://vcita.github.io/developers-hub/entities/dashboard/staffWidgetsBoard.json"
+            }
+          }
+        },
+        "required": [
+          "staffWidgetsBoards"
+        ]
+      },
+      "SectionDto": {
+        "type": "object",
+        "properties": {
+          "code_name": {
+            "type": "string",
+            "description": "The section code name"
+          },
+          "widgets": {
+            "description": "The sections widgets",
+            "allOf": [
+              {
+                "$ref": "#/components/schemas/SectionWidgetDto"
+              }
+            ]
+          }
+        },
+        "required": [
+          "code_name",
+          "widgets"
+        ]
+      },
+      "SectionWidgetDto": {
+        "type": "object",
+        "properties": {
+          "uid": {
+            "type": "string",
+            "description": "The widget uid"
+          },
+          "dimentions": {
+            "description": "The sections widgets",
+            "allOf": [
+              {
+                "$ref": "#/components/schemas/CustomizedWidgetDimentionsDto"
+              }
+            ]
+          }
+        },
+        "required": [
+          "uid"
+        ]
+      }
+    }
+  },
+  "paths": {
+    "/v3/widgets": {
```

Comment:
this path is not valid. you need a domain.
I suggest to have it in the same domain as apps as this is part of the apps framework IMO. I need the approval of Tal here, but I suggest:
 /v3/apps/widgets
 If you agree, you need to move all of the entities to the apps folder